### PR TITLE
Add support for the Official Terraria Wiki

### PR DESCRIPTION
**Pull request by RobbyCBennett**
terraria.wiki.gg

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -18714,6 +18714,22 @@ IGNORE INLINE STYLE
...
+terraria.wiki.gg
+
+CSS
+:root {
+    --darkreader-bg--theme-content-background: transparent;
+    --darkreader-bg--ooui--menuselect-background-color: #111111;
+}
+body, .navbox, #mw-head, #footer, #simpleSearch #searchButton, #simpleSearch #searchButton:hover, #searchInput:hover ~ #searchButton, #searchInput:focus ~ #searchButton, #mw-panel .portal, #mw-panel #p-logo + .portal {
+    background: none;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:18726]: What does this fix? The backgrounds are not breaking anything for Dark Reader.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -18714,6 +18714,22 @@ IGNORE INLINE STYLE
...
+.suggestions {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:18727]: Where can I find this element?

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -18714,6 +18714,22 @@ IGNORE INLINE STYLE
 
 ================================
 
+terraria.wiki.gg
+
+CSS
+:root {
+    --darkreader-bg--theme-content-background: transparent;
+    --darkreader-bg--ooui--menuselect-background-color: #111111;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:18722]: IMO the generated colors seems to be correct, given the non-modified site's colors.

**Review by Gusted — CHANGES_REQUESTED** [review]
Hi, thank you for your PR. However I currently don't see what this PR is trying to fix on this site, please explain these changes.

**Comment by RobbyCBennett** [comment]
Light mode:
light


Default dark mode (dynamic theme):
dark

The default dark mode is worse than light mode. The background is still white, and the tabs at the top become unreadable (Page, Discussion, Read, View source, and History)

My fix:
modified

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (RobbyCBennett) [reply src/config/dynamic-theme-fixes.config:18726]: See my comment below. This gets rid of the mid brown background colors and images. The mid brown has a value of 35% brightness. It's incredibly bright compared to the rest of the web. This also gets rid of the white background image.

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (RobbyCBennett) [reply src/config/dynamic-theme-fixes.config:18722]: `--darkreader-bg--theme-content-background: transparent;`
This is to get rid of a lot of the mid-brown background colors. Here's an alternative that has the same functionality.

```css
.mw-body, .infocard, .infocard .main-heading .hgroup {
    background: none;
}
```

`--darkreader-bg--ooui--menuselect-background-color`
This change is for the popup in when typing in the search bar to make it match the rest of the site background.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:18726]: Removing them is not the solution in this situation, you are better of by using this CSS rule instead:

```
    filter: brightness(50%) sepia(40%) !important;
```

To reduce the brightness of the image.

**Pull request closed without merging**